test/demo/index.js

```diff
@@ -1,5 +1,5 @@
 var EVENTS = ['start', 'stop', 'pause', 'resume']
-var TYPES = ['audio/webm', 'audio/ogg', 'audio/wav']
+var TYPES = ['audio/webm', 'audio/ogg', 'audio/wav', 'audio/mpeg']
```

Comment:
I think we need to revert this change since we reverted `isTypeSupported`

Reply:
done